### PR TITLE
Update security-compliance-overview.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/3539
Fixed link

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/security-compliance-overview.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ The following figure indicates the ingestion flow of Teams Meetings and calling
 
 When it comes to information protection capabilities, Office 365 subscriptions and the associated standalone licenses will determine the available feature set.
 
-For information on determining the licensing needs to implement features for security and compliance, please review: [Licensing for Office 365](https://docs.microsoft.com/office365/servicedescriptions/office-365-platform-service-description/office-365-securitycompliance-center).
+For information on determining the licensing needs to implement features for security and compliance, please review [Licensing for Office 365 or Microsoft 365](http://download.microsoft.com/download/8/7/7/877B1713-671E-43AA-BB79-AF8478C64AFF/Licensing-Microsoft-365.pdf)
```
Comment (JohanFreelancer9) [inline Teams/security-compliance-overview.md]:
```suggestion
For information on determining the licensing needs to implement features for security and compliance, please review [Licensing for Office 365 or Microsoft 365](http://download.microsoft.com/download/8/7/7/877B1713-671E-43AA-BB79-AF8478C64AFF/Licensing-Microsoft-365.pdf).
```

**Comment by scanum** [comment]
@MicrosoftHeidi PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MicrosoftHeidi** [comment]
The link needs to be https, not http. Fix that, and you should be able to publish it.

File: Teams/security-compliance-overview.md [hunk 2]
```diff
@@ -143,7 +143,7 @@ The following figure indicates the ingestion flow of Teams Meetings and calling
 
 When it comes to information protection capabilities, Office 365 subscriptions and the associated standalone licenses will determine the available feature set.
 
-For information on determining the licensing needs to implement features for security and compliance, please review: [Licensing for Office 365](https://docs.microsoft.com/office365/servicedescriptions/office-365-platform-service-description/office-365-securitycompliance-center).
+For information on determining the licensing needs to implement features for security and compliance, please review [Licensing for Office 365 or Microsoft 365](http://download.microsoft.com/download/8/7/7/877B1713-671E-43AA-BB79-AF8478C64AFF/Licensing-Microsoft-365.pdf).
```
Comment (JohanFreelancer9) [inline Teams/security-compliance-overview.md]:
```suggestion
For information on determining the licensing needs to implement features for security and compliance, please review [Licensing for Office 365 or Microsoft 365](https://download.microsoft.com/download/8/7/7/877B1713-671E-43AA-BB79-AF8478C64AFF/Licensing-Microsoft-365.pdf).
```

**Pull request merged**